Assets/Plugins/Source/Editor/Configs/SteamConfig.cs

```diff
@@ -20,14 +20,16 @@
 * SOFTWARE.
 */
 
-using System;
-using System.Collections.Generic;
 
-namespace PlayEveryWare.EpicOnlineServices
+
+namespace PlayEveryWare.EpicOnlineServices.Editor.Config
```

Comment:
Shouldn't it also be used in Windows (non-editor) when running under steam? If that is what is supposed to be happening, it may need to be fixed there because it's not referenced _anywhere_ in non-editor code. 

Regardless, I have restored it to the previous namespace for the time being.

Reply:
On windows the steam config is read in the GFX plugin, so this code won't be hit when running on Windows standalone builds.